src/algebra/jordan/basic.lean

```diff
@@ -0,0 +1,262 @@
+/-
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import algebra.ring.basic
+import algebra.lie.of_associative
+import data.real.basic
+import linear_algebra.basic
+
+/-!
+# Jordan algebras
+
+Let `A` be a non-associative algebra (i.e. a module equipped with a bilinear multiplication
+operation). Then `A` is said to be a (commutative) Jordan algebra if the multiplication is
+commutative and satisfies a weak associativity law known as the Jordan Identity: for all `a` and `b`
+in `A`,
+```
+(a * b) * a^2 = a * (b * a^2)
+```
+i.e. the operators of multiplication by `a` and `a^2` commute. Every associative algebra can be
+equipped with a second  multiplication making it into a commutative Jordan algebra.
+Jordan algebras arising this way are said to be special. There are also exceptional Jordan algebras
+which can be shown not to be the symmetrisation of any associative algebra. The three by three
+matrices of octonians is the canonical example.
+
+Commutative Jordan algebras were introduced by Jordan, von Neumann and Wigner
+([jordanvonneumannwigner1934]) as a mathematical model for the observables of a quantum mechanical
+physical system (for a C*-algebra the self-adjoint part is closed under the symmetrised Jordan
+multiplication). Jordan algebras have subsequently been studied from the points of view of abstract
```

Comment:
Am I right in thinking that this was not ultimately a very fruitful point of view?

If so I think it might be better to emphasise the reasons Jordan algebras are still studied rather than the original motivation which, I believe, wasn't very fruitful.

Reply:
Yes, this is explained at some length in McCrimmon's "A Taste of Jordan Algebras".  I could expand the entry a bit to explain this. Most accounts of the theory seem to use the Jordan, von Neumann and Wigner paper as a starting point.

Reply:
I don't know the theory of Jordan algebras well enough to say much but maybe we could go with something like:

"Although Jordan algebras were originally introduced to axiomatise observables in quantum mechanics, this did not prove fruitful. So far their major successes have been within their own field of algebra, especially their role in Zelmanov's solution of the restricted Burnside problem."

Reply:
I can't profess to understand the physics, but my understanding is that, although Zelmanov proved that there were no infinite dimensional simple exceptional Jordan algebras, which put an end to Jordan's original idea, since the 1970s Jordan algebras have been of relevance to quantum mechanics through something called the "operational approach to axiomatic statistical physical theories". In differential geometry unital JB*-algebras are equivalent to tube domains. Ultimately the real star is the Jordan triple product rather than the binary product though.

Reply:
I think you have stronger feelings here than I do so I'll let you decide what to write.

For me the job of a comment like this is to answer the question "why should I care about this structure?" and I was hoping we could find something short that answered this.

Reply:
I've pushed a commit with a list of other areas that Jordan algebras are connected to, and references to books which explain these things much better than I can.

I think Jordan et al were looking for some infinite dimensional structures completely divorced from associative algebras, and were disappointed when they didn't find them. Subsequently it has transpired that new, physically meaningful, insights can be obtained by ignoring the associative multiplication and working with the Jordan one instead. Judging from Bertram's [review](http://wolfgang.bertram.perso.math.cnrs.fr/siam.pdf) of McCrimmon's book, I'm not the only person to have been puzzled by this.